packages/release-mastermind/src/contexts/methods/conventions.ts

```diff
@@ -8,93 +1,145 @@
+				conditions.push({
+					type: "titleMatches",
+					pattern: `/^${pattern}(\\(.*\\))?:/i`
+				})
+			})
+			if (convention.contexts) {
+				convention.requires++
+				convention.contexts.forEach((pattern) => {
+					conditions.push({
+						type: "titleMatches",
+						pattern: `/\\(.*${pattern}.*\\):/i`
+					})
+				})
+			}
+			convention.failedComment =
+				`Semantic Conditions failed - Please title your ${
+					this.curContext.type == "pr" ? "pull request" : "issue"
+				} using one of the valid options:\r\n\r\n Types: ` +
+				semantic.join(", ") +
+				(convention.contexts
+					? `\r\n\r\n Contexts: ${convention.contexts.join(", ")}`
+					: "")
+			convention.conditions = conditions
+		}
+		if (evaluator.call(this, convention, this.context.props)) {
+			successful++
+		} else {
+			failedMessages.push(convention.failedComment)
+		}
+	})
 
-  if (required > successful) {
-    failedMessages.forEach(fail => core.setFailed(fail))
-    !this.dryRun && createConventionComment.call(this, false, failedMessages)
-    return false
-  }
-  log(
-      LoggingLevels.info,
-      `All conventions successfully enforced. Moving to next step`
-  )
-  !this.dryRun && createConventionComment.call(this, true)
-  return true
+	if (required > successful) {
+		failedMessages.forEach((fail) => core.setFailed(fail))
+		!this.dryRun && createConventionComment.call(this, false, failedMessages)
+		return false
+	}
+	log(
+		LoggingLevels.info,
+		`All conventions successfully enforced. Moving to next step`
+	)
+	!this.dryRun && createConventionComment.call(this, true)
+	return true
 }
 
 async function createConventionComment(
-  this: Issues | PullRequests | Project,
-  success: boolean,
-  failMessages?: string[]
+	this: Issues | PullRequests | Project,
+	success: boolean,
+	failMessages?: string[]
 ) {
-  if (!this.config.enforceConventions) return
-  let prefix: string = `<!--releaseMastermind: Conventions-->${
-      this.config.enforceConventions.commentHeader || ''
-    }\r\n\r\n`,
-    suffix: string = `\r\n\r\n----------\r\n\r\nThis message will be automatically updated when you make this change\r\n\r\n${
-      this.config.enforceConventions.commentFooter || ''
-    }`,
-    body: string = prefix + failMessages?.join('\r\n\r\n') + suffix,
-    commentList
-  if (this.curContext.type !== 'pr') {
-    commentList = await this.util.api.issues.comments.list(
-      this.curContext.context.props.ID
-    )
-  } else {
-    commentList = await this.util.api.pullRequests.reviews.list(
-      this.curContext.context.props.ID
-    )
-  }
-  let previousComment: number | undefined
-  if (commentList) {
-    commentList.forEach((comment: any) => {
-      if (comment.body.includes(prefix) && comment.state !== 'DISMISSED')
-        previousComment = comment.id
-    })
-  }
-  this.util.respond(this, success, previousComment, body)
+	if (!this.config.enforceConventions) return
+	let prefix = `<!--releaseMastermind: Conventions-->`,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
	let prefix = "<!--releaseMastermind: Conventions-->",
```